### ISSUE TITLE
Import Errors in Swift

### ISSUE THREAD

**Issue by AnthonyMDev** [tombenner/nui#355]
I'm getting an error when importing this library in Swift `'NUIParse.h' file not found.

This is fixed by changing all import statements from `#import "NUIParse.h"` to `#import <NUIParse/NUIParse.h>`.

Does this change break anything for Objective-C users?

**Comment by Stunner** [comment]
No, it shouldn't cause any problem for Objective-C users. If you could submit a pull request with these changes I'd be happy to merge it in.

**Comment by AnthonyMDev** [comment]
Handling that now 👍

**Comment by AnthonyMDev** [comment]
It looks like this is actually fixed on `master` using:

````swift
#if __has_include(<NUIParse/NUIParse.h>)
#    import <NUIParse/NUIParse.h>
#else
#    import "NUIParse.h"
#endif
````

**Comment by Stunner** [comment]
@timbodeit Your thoughts on this? Considering we are both awfully inactive on this project these days, it may make sense to roll out a 0.5.x release to tie people over until we can get 0.6.0 ready and working.
